### ISSUE TITLE
MD013 : Issue with MarkdownLint Configuration Ignoring Line Length Exceeding 80 Words

### ISSUE THREAD

**Issue by SOHAN-cyber** [DavidAnson/markdownlint#1136]
I've encountered while configuring MarkdownLint to ignore line length exceeding 80 words. Despite following the official documentation's guidance, the line length issue persists, and I seek assistance in resolving it.

Here's the current configuration I've set up:
```
 ---
# MarkdownLint YAML configuration file
#   https://github.com/DavidAnson/markdownlint-cli2?tab=readme-ov-file#markdownlint-cli2yaml
#   https://github.com/DavidAnson/markdownlint-cli2/blob/main/test/markdownlint-cli2-yaml-example/.markdownlint-cli2.yaml

config:
  # MD013/line-length: https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md013.md
  line-length:
    tables: false # Do not complain about wide tables generated by Terraform Docs

# Fix any fixable errors
fix: true

# Define glob expressions to ignore
ignores:
  # MarkdownLint detects this file as a '.markdown' file
  - "**/.markdownlint-cli2.yaml"
```

I've integrated this configuration into the pre-commit setup as follows:

```
  - repo: https://github.com/DavidAnson/markdownlint-cli2
    rev: v0.12.1
    hooks:
      - id: markdownlint-cli2
        args:
          - extern/make/.config/.markdownlint-cli2.yaml
```

Despite these settings, MarkdownLint continues to flag lines exceeding 80 characters, contrary to my intent to ignore this rule. I'm uncertain where the misconfiguration lies and would appreciate any guidance or adjustments you could provide to resolve this issue effectively.

**Comment by DavidAnson** [comment]
The pre-commit configuration above is using "args" to say the YAML configuration file should be linted (and nothing else). You probably want to use the --config option and pass a glob like "**/*.md". Once that problem is fixed, the "ignores" override you have should no longer be necessary and the other configuration should be applied as expected.

**Comment by DavidAnson** [comment]
This tool shows what it's doing at the beginning of its output. You can see that in your bottom snippet as something that looks like this: `Finding: --config=extern/make/.config/.markdownlint-cli2.yaml .gitlab/issue_templates/Default.md ...`

That tells us the syntax for "--config" was wrong because "=" should not be used there. The pre-commit syntax you show above looks reasonable to me, but I do not use pre-commit. In my own smoke test for pre-commit integration, I use this syntax and it seems to behave as expected: `args: [--config, configuration/.markdownlint-cli2.jsonc]`
